docs/release-notes/v2.26/index.md

```diff
@@ -0,0 +1,150 @@
+---
+date: "2020-11-12"
+version: "2.26.0"
+---
+
+# [v2.26](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.26.0-next.0...gatsby@2.27.0-next.0) (November 2020)
+
+---
+
+Welcome to `gatsby@2.26.0` release (November 2020).
+Key highlights of this release:
+
+- [New Image Plugin](#gatsby-plugin-image010-beta) - access static images without GraphQL, high Lighthouse scores again
+- [File System Route API](#file-system-route-api) - create pages from your data using filename conventions
+- [New Release Process](#new-release-process) - more stability and clarity around Gatsby releases
+- [gatsby-source-contentful v4.0](#gatsby-source-contentful400) - new RichText implementation, performance improvements
+
+Other notable changes:
+
+- [gatsby-plugin-mdx](#gatsby-plugin-mdx140) - performance improvements
+- [Enable compression in the dev server](#enable-compression-in-the-dev-server) - speeds up remote previews
+
+Sneak peek to next releases:
+
+- [Queries on Demand](#queries-on-demand) - improves `gatsby develop` bootup time
+- [Experimental: Lazy images in develop](#experimental-lazy-images-in-develop)
+- [Documentation Reorganization](#documentation-reorganization)
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know
+if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+### New Release Process
+
+We are evaluating a new process that should bring more stability and clarity
+to Gatsby releases.
+
+### File System Route API
+
+This new API enables you to create pages from your GraphQL data using filename conventions,
+e.g. to create individual blog post pages for your blog.
+
+Some examples:
+
+- `src/pages/products/{Product.name}.js` will generate pages for all products with a path like `/products/burger`
+- `src/pages/products/{Product.fields__sku}.js` will generate pages for all products with a path like `/products/001923`
+
+See [Full Documentation](https://www.gatsbyjs.com/docs/file-system-route-api/),
+[examples](https://github.com/gatsbyjs/gatsby/tree/master/examples/route-api)
+and related PR [#27424](https://github.com/gatsbyjs/gatsby/pull/27424).
+
+### Enable compression in the dev server
+
+This lowers the amount of data transferred for one site especially in remote situations.
+Most of that drop was from the commons.js bundle.
+
+See PR [#27948](https://github.com/gatsbyjs/gatsby/pull/27948) for details.
+
+## gatsby-plugin-image@0.1.0 (beta)
+
+New image plugin to replace `gatsby-image`, which greatly improves performance (Lighthouse 💯 again)
+and adds easy static images (no GraphQL). Part of it is also a new, simpler API for `gatsby-transformer-sharp`.
```

Comment:
I think this would really benefit from a code sample to show how simple it is

Reply:
Added a bit more details and examples